### PR TITLE
[Event] added kcd chennai talk to events page

### LINKED ISSUE #4388: [Event] Add KCD Chennai Talk to Events page
Please add the following event to layer5.io/events.

**Event:**
- Event Title: KCD Chennai 2023
- Event Link: https://www.kcdchennai.in/
- Talk Description: Multiplayer Kubernetes: GitOps with Friends

Join Pranav Singh at KCD Chennai 2023 to witness firsthand how Meshery revolutionizes Kubernetes operations, enabling seamless orchestration across multiple environments made possible by GitOps principles and multi-user collaboration..

- Talk Link: 
- Deck: 
- Video: 

---
<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Chetan-Managavi**
**Description**

This PR fixes #4388. Added Thumbnail and event details of KCD chennai to events page.
KCD-Chennai-Event

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 634ad0f06836eecaf25fa549c35bea10375c608f at: https://648d641772e8805f386e0351--layer5.netlify.app

File: src/collections/events/kcd-chennai-2023/index.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "KCD Chennai 2023"
+date: 2023-07-22
+thumbnail: ./kcd_chennai.webp
+darkthumbnail: ./kcd_chennai.webp
```
Comment (leecalcote) [inline src/collections/events/kcd-chennai-2023/index.mdx:5]: Thanks for using a .webp, @Chetan-Managavi 👍

File: src/collections/events/kcd-chennai-2023/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+eurl: https://www.kcdchennai.in/
+type: Event
+published: true
+product: Meshery
```
Comment (leecalcote) [inline src/collections/events/kcd-chennai-2023/index.mdx:9]: Could you toss in "MeshMap" in addition to "Meshery"?

**Comment by l5io** [comment]
🚀 Preview for commit 0456d2cb5c40119c9dd2fef523b66708ee251bd5 at: https://648e8e1411e1f6753265007e--layer5.netlify.app

File: src/collections/events/kcd-chennai-2023/index.mdx (hunk 2 above)
Comment (Chetan-Managavi) [reply src/collections/events/kcd-chennai-2023/index.mdx]: @leecalcote  check out the changes in the latest commit https://github.com/layer5io/layer5/commit/0456d2cb5c40119c9dd2fef523b66708ee251bd5

File: src/collections/events/kcd-chennai-2023/index.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: "KCD Chennai 2023"
+date: 2023-07-22
+thumbnail: ./kcd_chennai.webp
+darkthumbnail: ./kcd_chennai.webp
+eurl: https://www.kcdchennai.in/
+type: Event
+published: true
+product: MeshMap , Meshery
```
Comment (leecalcote) [inline src/collections/events/kcd-chennai-2023/index.mdx:9]:
```suggestion
product: MeshMap, Meshery
```

**Review by leecalcote — APPROVED** [review]
Thanks, @Chetan-Managavi 👍

**Comment by theBeginner86** [comment]
Thanks @Chetan-Managavi 🚀

**Pull request merged**